neptune/exceptions.py

```diff
@@ -13,18 +13,54 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import warnings
+
 from neptune import envs
 
+STYLES = {'h1': '\033[95m',
```

Comment:
Here is a nice comment on how to detect if a terminal supports ANSI color codes. 
https://stackoverflow.com/questions/7445658/how-to-detect-if-the-console-does-support-ansi-escape-codes-in-python

Also, as we talked offline, an ENV-based switch would be nice.

For what it's worth, I think it will work in most cases (I checked my own terminal and Jupyter notebook/lab). I'm mostly worried about Windows, TBH - haven't check this yet.

Reply:
Hm, ok, so unfortunately, I checked and on Windows I get get this (both in the standard CMD and in PowerShell):
![image](https://user-images.githubusercontent.com/13830066/95622379-a757d480-0a73-11eb-8487-e3644c0dbba3.png)

So it seems that we need to do some detecting before using colors.

Reply:
Done.

For now it just gives no collors to Windows users but if we figure it out we can change it later.